Upstream recent release `typing-extensions==4.6.0` is conflicting with `pydantic==1.9.0`

Yesterday, [`typing-extensions` 4.6.0 was released upstream](https://github.com/python/typing_extensions/blob/main/CHANGELOG.md#release-460-may-22-2023), this is a development upstream dependency that claims to always be backwards compatible, but it ended up conflicting with `pydantic==1.9.0` which does some metaprogramming with type inspection, this only impacts testing, either we need to upgrade python or pin `typing-extensions==4.5.0`. Pinning `typing-extensions==4.5.0` would be the easiest, but let's see if it's already worth also upgrading pydantic.

```
❯ python3 setup.py test
running test
ImportError while loading conftest '/home/viniarck/repos/kytos/tests/conftest.py'.
tests/conftest.py:7: in <module>
    from kytos.core import Controller
kytos/core/__init__.py:4: in <module>
    from kytos.core.auth import authenticated
kytos/core/auth.py:27: in <module>
    from kytos.core.user import HashSubDoc, UserDoc, UserDocUpdate
kytos/core/user.py:42: in <module>
    class UserDoc(DocumentBaseModel):
pydantic/main.py:204: in pydantic.main.ModelMetaclass.__new__
    ???
pydantic/fields.py:488: in pydantic.fields.ModelField.infer
    ???
pydantic/fields.py:419: in pydantic.fields.ModelField.__init__
    ???
pydantic/fields.py:534: in pydantic.fields.ModelField.prepare
    ???
pydantic/fields.py:638: in pydantic.fields.ModelField._type_analysis
    ???
/usr/lib/python3.9/typing.py:852: in __subclasscheck__
    return issubclass(cls, self.__origin__)
E   TypeError: issubclass() arg 1 must be a class
```

Thanks for reporting this @italovalcy

--- a/requirements/dev.txt
+++ b/requirements/dev.txt
@@ -446,7 +446,7 @@ traitlets==5.3.0
     #   ipython
     #   kytos
     #   matplotlib-inline
-typing-extensions>=4.0.1
+typing-extensions==4.5.0
     # via
     #   -r requirements/run.txt
     #   astroid

--- a/requirements/run.txt
+++ b/requirements/run.txt
@@ -161,7 +161,7 @@ traitlets==5.3.0
     # via
     #   ipython
     #   matplotlib-inline
-typing-extensions>=4.0.1
+typing-extensions==4.5.0
     # via
     #   janus
     #   jsonschema-spec

--- a/setup.py
+++ b/setup.py
@@ -232,7 +232,8 @@ setup(name='kytos',
           'pycodestyle==2.10.0',
           'yala==3.2.0',
           'tox==3.28.0',
-          'virtualenv==20.21.0'
+          'virtualenv==20.21.0',
+          'typing-extensions==4.5.0'
       ]},
       cmdclass={
           'clean': Cleaner,

